run-benchmark.sh

```diff
@@ -145,6 +151,11 @@ BASEDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 BASEDIR=$(fixPath $BASEDIR)
 #debug basedir ends in / $BASEDIR
 
+# setup VERSION
+VERSION=$(cat ${BASEDIR}VERSION)
+VERSION_DATE=$(date -r ${BASEDIR}VERSION)
+echo "##### Splice Benchmark Harness version: $VERSION ($VERSION_DATE)"
```

Comment:
this seems like a 'verbose' printout, instead of 'echo', use 'message', which only prints if the -v flag is passed in

Reply:
I need this line in Jenkins console output unconditionally. At some point I'll remove -V -D from Jenkins builds but I still want to know which version is used.